x-pack/filebeat/module/suricata/eve/_meta/fields.yml

```diff
@@ -3,14 +3,6 @@
   description: >
     Fields exported by the EVE JSON logs
   fields:
-    - name: src_ip
-      type: ip
-    - name: dest_ip
-      type: ip
     - name: flow_id
       type: keyword
-    # Geo Points should be the ECS fields, but no way to config from here.
-    - name: src_geo.location
-      type: geo_point
-    - name: dest_geo.location
-      type: geo_point
```

Comment:
You removing these fields, does that mean you're no longer keeping the original Suricata EVE event under `suricata.eve`? How are you approaching this?

The basics are easy enough to map to ECS, like the IP, port, geo etc. But there's a lot of data that won't be easy to map to ECS at the moment. The stats events, even alert events I'm not sure we have a clear 1:1 mapping for all the fields.

Reply:
No, I was planing to keep the original suricata event verbatim. Removing `src_ip` and `dest_ip` was a mistake, good catch.

I understand that `src_geo` and `dest_geo` were created by the pipeline and not included in the original event, so now that it's storing this information into source(_ecs) and destination, I'm removing them from suricata.eve. Makes sense?

Reply:
I was planning to keep the original suricata event verbatim. Removing src_ip and dest_ip was a mistake, good catch.

I understand that src_geo and dest_geo were created by the pipeline and not included in the original event, so now that it's storing this information into source(_ecs) and destination, I'm removing them from suricata.eve. Makes sense?

Reply:
Yes having the geo only in the ECS fields makes total sense.

I had initially defined them there because of the limitation of suricata/eve/_meta/fields.yml only being able to define field types under `suricata.eve.`. So I agree the `geo` points can be removed from there.